.prettierignore

```diff
@@ -10,4 +10,14 @@ build
 tests
 webdriver
 *.yml
-*.json
\ No newline at end of file
+*.json
+docs/web-apps/automated-testing/cucumberjs-playwright/yaml.md
+docs/web-apps/automated-testing/cypress/yaml
+docs/web-apps/automated-testing/cypress/yaml.md
```

Comment:
```suggestion
```

Reply:
@spider-sauce There are no code blocks in docs/web-apps/automated-testing/cypress/yaml.md --- only the docs/web-apps/automated-testing/cypress/yaml folder should be added to .prettierignore.

Reply:
I removed these files from the ignore list and just added the `"embeddedLanguageFormatting": "off"` to `.prettierrc`. I just tested it on one of the yaml docs and it worked.